exec/java-exec/src/test/java/org/apache/drill/exec/store/bson/TestBsonRecordReader.java

```diff
@@ -272,6 +276,9 @@ public static void cleanUp() {
     } catch (Exception e) {
 
     }
+
+    buffer.close();
```

Comment:
buffer.release()?

Reply:
Currently close() just calls release(). I will update to use release(), but in general what is the best practice regarding when to call close() and when to call release()?

Reply:
IMO `close()` should be used within try-with-resources only (no explicit calls to `close()`).